test/quic_multistream_test.c

```diff
@@ -962,21 +1187,106 @@ static int run_script(const struct script_op *script, int free_order)
             }
             break;
 
+        case OPK_NEW_THREAD:
+            {
+#if !defined(OPENSSL_THREADS)
+                TEST_error("threading not supported");
```

Comment:
How is this supported to work with no threads? I do not see any script exclusion for these multithread test scripts below?